core/survey/survey.js

```diff
@@ -25,7 +25,16 @@ import * as SurveyRefDataIndex from './_survey/surveyRefDataIndex'
  * @param {object} [params.otherProps=null] - Other props to set.
  * @returns {object} The newly created survey info object.
  */
-export const newSurvey = ({ ownerUuid, name, label = null, languages, published = false, draft = true, ...rest }) => ({
+export const newSurvey = ({
+  ownerUuid,
+  name,
+  label = null,
+  languages,
+  published = false,
+  draft = true,
+  template = false,
```

Comment:
this must be added to arena-core as well

Reply:
uhm, ok, right